Source/SwiftLintBuiltInRules/Rules/Lint/CaptureVariableRule.swift

```diff
@@ -158,8 +158,11 @@ struct CaptureVariableRule: AnalyzerRule, CollectingRule {
         file.declaredVariables(compilerArguments: compilerArguments)
     }
 
-    func validate(file: SwiftLintFile, collectedInfo: [SwiftLintFile: Self.FileInfo],
-                  compilerArguments: [String]) -> [StyleViolation] {
+    func validate(
+        file: SwiftLintFile,
+        collectedInfo: [SwiftLintFile: Self.FileInfo],
+        compilerArguments: [String]
+    ) -> [StyleViolation] {
```

Comment:
Position of the return type is inconsistent here. Should it go to the end of the line above?